RevenueCat/Scripts/CustomerInfo.cs

```diff
@@ -7,14 +7,17 @@
 
 public partial class Purchases
 {
-    /*
-     * CustomerInfo encapsulate the current status of subscriber. 
-     * Use it to determine which entitlements to unlock, typically by checking 
-     * ActiveSubscriptions or via LatestExpirationDate. 
-     * 
-     * Note: All DateTimes are in UTC, be sure to compare them with 
-     * DateTime.UtcNow
-     */
+    ///
+    /// <summary>
+    /// CustomerInfo encapsulates the current status of subscriber. 
+    /// Use it to determine which entitlements to unlock, typically by checking 
+    /// ActiveSubscriptions or via LatestExpirationDate.
+    /// </summary> 
+    /// 
+    /// <remarks>
+    /// All DateTimes are in UTC, be sure to compare them with <c>DateTime.UtcNow</c>
+    /// </remarks>
+    ///
```

Comment:
for some of these I just added a docstring to the class itself, not its members. baby steps.